tests/Tests/E2e/CreateStaffTest.php

```diff
@@ -33,7 +33,7 @@ public function check_add_user(): void
         $openEmrPage = $this->e2eBaseUrl;
         $client = static::createPantherClient(['external_base_uri' => $openEmrPage]);
         $lp = new LoginPage($client, $this);
-        $mp = $lp->login('admin', 'pass');
+        $mp = $lp->login('admin', 'password123');
```

Comment:
This is failing all e2e testing since it needs to be the actual password of the admin user.